frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -755,7 +757,12 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
         ) : (
           containers.map((container: any, i: number) => (
             <SectionBox key={i} outterBoxProps={{ pt: 1 }}>
-              <ContainerInfo container={container} status={statuses[container.name]} />
+              <ContainerInfo
+                podname={resource?.metadata.name}
+                namespace={resource?.metadata.namespace}
```

Comment:
Should we just pass the resource? And then within it use what we need from it instead of passing two props?

Reply:
updated